src/i18n/locale/en.json

```diff
@@ -1529,4 +1531,4 @@
   "pages.returnHome": "Return Home",
   "pages.serviceunavailable": "Service Unavailable",
   "pages.somethingwentwrong": "Something Went Wrong"
-}
+}
```

Comment:
This newline shouldn't be removed (it's maybe the reason why the github actions are failing)

Reply:
Ah yes, good catch. I think that was my file formatter updating it.